### ISSUE TITLE
Refactor .pl` source to src/

### ISSUE THREAD

**Issue by ssardina** [SWI-Prolog/packages-jpl#78]
All the Java and C code has been refactored into Maven-style structure within `/src/`, but the Prolog code  `.pl` remains in the root dir.

Would be good to move `jpl.pl`, and even `jpl_config.pl` to `/src/main/prolog/`

We should not move `test_jpl.pl` because the whole SWIPL is set-up to find the unit testing in the root of every package. That should stay there

**Comment by JanWielemaker** [comment]
I'm not really against, but not much in favor either.   All packages have very little organization in terms of directories.  Prolog doesn't need it, it is one (and a little helper) file that gets copied to the library.   Moving it mainly means regressing risks.

**Comment by ssardina** [comment]
OK, I take your point. Indeed that is why I did not do it before when I restructured the Java and C code. I am now confident I can handle the copying with CMAKE.

But you are right, maybe there is not much to win and something to lose here, and in any case I am not moving the `test_jpl.pl` to adhere to the structure on the other packages. 

So, point taken, I will drop this. Thanks for commenting your thoughts.
